documentation/crash/step2/using_llnode.md

```diff
@@ -1 +1,52 @@
-// TODO
+# Using `llnode` for post-mortem debugging
+
+You probably already experienced bugs that only happen in production. You'd like to reproduce it and step through instructions, but it will imply stopping your Node.js process and certainly degrade the user experience.
+
+The purpose of this guide is to help you to find route causes
+of your application crashes in production.
+
+Before jumping into the code, let's clarify a few core concepts
+that will helps you to go thoroughly through this tutorial.
+
+## Concepts
+
+### What is post-mortem debugging?
```

Comment:
I don't think this is needed. Even being important, I don't see beginners going to this document as a first reference. I see this document as a tutorial on a specific workflow, in this case, using llnode to diagnose application _crash_.

So, for instance, I'd replace the sections of `What is ***` to `Finding function using llnode` or `Get crash report using llnode`

Does that make sense?

Reply:
> I don't think this is needed. Even being important, I don't see beginners going to this document as a first reference. I see this document as a tutorial on a specific workflow, in this case, using llnode to diagnose application crash.

I agree to delete the `What is post-mortem debugging`; it seems too broad. But for the two others (core dumps & lldb) it could be relevant IMO. What we could do is invite people that are familiar with these concepts to go to the next chapter. WDYT?

Reply:
SGTM